app/views/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-8">
+      <h1>Edit an animal</h1>
+      <%= render 'form', animal: @animal %>
```

Comment:
you can add a _form.html.erb file and paste the simple_form from your new page there and then do <%= render 'form', animal: @animal %> in your new.html.erb and edit.html.erb